scripts/compatibility-test/readme.md

```diff
@@ -0,0 +1,38 @@
+﻿
+## Steps to configure & run compatibility test.
+
+Pre-requisites:
+   1. S3Server should be up and running 
+   
+Steps:
+
+1. Navigate to the compatibility-test script folder
+   ```
+      cd cortx-s3server/scripts/compatibility-test
+   ```
+
+2. create s3test.conf for the corresponding integration. Refer sample template for ceph and plunk integrations in the respective folders. Two s3 account credentials need to be updated in the configurations
```

Comment:
minor spelling mistake pls correct it as "splunk "

Reply:
Fixed the typo